specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzeconversations-authoring.json

```diff
@@ -2235,6 +2969,16 @@
       },
       "x-ms-client-name": "JobStatus"
     },
+    "ConversationalAnalysisAuthoringLoadSnapshotJobState": {
+      "description": "Represents the state of loading a snapshot job.",
+      "type": "object",
+      "allOf": [
```

Comment:
If this is only an extension of the generic (authoring) job state, how does one get the final value? See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs for our current standards.

Reply:
Same as the other comment, the final value is the inline status when it reaches a terminal state.